cgmanifest.json

```diff
@@ -35,6 +35,15 @@
                "repositoryUrl":"https://github.com/Microsoft/GSL.git"
             }
          }
+      },
+       {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"d846fe50a3f0bb7767c7e087a05f4be95f4da0ec",
+               "repositoryUrl":"https://github.com/Microsoft/GSL.git"
+            }
+         }
```

Comment:
dup

Reply:
We had two registered previously with diff commit hashes. Should we just have one with the latest?

Reply:
cee3125 is the one in use.

Reply:
Fixed.